### ISSUE TITLE
Create MVP

### ISSUE THREAD

**Issue by codingwithelijah** [amosproj/amos2022ss05-find-my-hearing-aid#19]
## User story
1. As a PO
2. I want / need to visualize a simple draft of the product idea
3. So that we can receive feedback from the customer quickly and easily

## Acceptance criteria
* Based on story map, create simple MVP
* Discuss MVP with whole team
* Show MVP to customer afterwards to get customer´s feedback

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by codingwithelijah** [comment]
Details regarding MVP: 

<img width="1196" alt="image" src="https://user-images.githubusercontent.com/74047429/167430186-8e968a2c-9baa-4dee-9d01-9d88cc2ae1b8.png">
